### PR TITLE
feat: create rich content menu

### PR DESCRIPTION
**Pull request by ces-huyDang**
create menu popup to show contents, add meta score block

CU-86ctwqu0m

### CODE REVIEW

File: web/modules/custom/blog/src/Controller/BlogController.php [hunk 1]
```diff
@@ -186,4 +186,10 @@ public function rate($score_info)
         $message = $this->blog_service->rating($score_info);
         return new JsonResponse($message);
     }
+
+    public function test()
```
Comment (ces-khanhhoang) [inline web/modules/custom/blog/src/Controller/BlogController.php:190]: Why name the function test?

File: web/modules/custom/blog/src/Controller/BlogController.php [hunk 2]
```diff
@@ -186,4 +186,10 @@ public function rate($score_info)
...
+    {
+        $test = $this->blog_service->getTaxonomyTerms('streaming_site');
```
Comment (ces-khanhhoang) [inline web/modules/custom/blog/src/Controller/BlogController.php:192]: as above

File: web/modules/custom/blog/src/Services/BlogService.php [hunk 3]
```diff
@@ -118,12 +118,22 @@ public function getPostInfo($node)
         $node_created = $node->getCreatedTime();
         $node_changed = $node->get('changed')->getValue()[0]['value'];
         $node_body = $node->get('body')->getValue()[0]['value'];
-        $node_thumbnail_uri = $this->getImagesSrc(
-            $node->get('field_thumbnail_image')
-                ->getValue(), "uri"
-        );
+        if ($node->get('type')->getValue()[0]['target_id'] === "posts") {
```
Comment (ces-khanhhoang) [inline web/modules/custom/blog/src/Services/BlogService.php:121]: add comment for this code logic

File: web/modules/custom/blog/src/Controller/BlogController.php (hunk 1 above)
Comment (ces-huyDang) [reply web/modules/custom/blog/src/Controller/BlogController.php]: Unnecessary function ~> Removed.

File: web/modules/custom/blog/src/Controller/BlogController.php (hunk 2 above)
Comment (ces-huyDang) [reply web/modules/custom/blog/src/Controller/BlogController.php]: Unnecessary function ~> Removed.

**Pull request closed without merging**